.github/workflows/qat.yml

```diff
@@ -15,160 +15,70 @@ on:
-          exit_code=$?
-          set -e # Enable exit on error
-          if [ $exit_code -ne 0 ]; then
-            echo "::error::Checkly checks failed. Please check the checkly-github-report.md in the summary for more details."
-            echo "checkly_failed=true" >> $GITHUB_OUTPUT
+          # GITHUB_HEAD_REF will only be set during a PR
+          if [[ "${GITHUB_HEAD_REF}" != "" ]]; then
+            # This is a pull request
+            BRANCH_NAME=${GITHUB_HEAD_REF#*/}
           else
-            echo "::notice::Checkly checks Passed! Please check the checkly-github-report.md in the summary for more details."
-            echo "checkly_failed=false" >> $GITHUB_OUTPUT
+            # This is the 'main' branch
+            BRANCH_NAME=${GITHUB_REF##*/}
           fi
-          cat checkly-github-report.md > $GITHUB_STEP_SUMMARY
-      - name: Deploy checks # Deploy checks to Checkly for debugging
-        # Only run if the pull request is merging into the main branch
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
-        id: deploy-checks
-        run: npx checkly deploy --config=$(find . -name checkly.config.ts) --force
-      - name: Add Checkly report comment to PR
-        uses: actions/github-script@v6
-        if: github.event_name == 'pull_request'
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            const fs = require('fs');
-            const report = fs.readFileSync('checkly-github-report.md', 'utf8');
-            const checklyFailed = ${{ steps.run-checks.outputs.checkly_failed }};
-            const checklyStatus = checklyFailed ? ':x: Some Playwright tests failed. Please review the Checkly Report to verify no breaking changes were introduced' : ':white_check_mark: All Playwright tests passed. Checkly Report';
 
-            const existingComment = await github.rest.issues.listComments({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo
-            });
+          # Set environment URLs based on the branch name
+          #
+          # The `version-hash` file is created by jenkins script when a branch is pushed
+          # https://github.com/penske-media-corp/pmc-jenkins-scripts/blob/8ee0244a8485b61cfa6d3f9af00f7e6a55ddb042/bin/push-pmcqa-repo.sh#L36-L38
+          # 
+          # Examples
+          #
+          # Feature branch example:
+          # ENVIRONMENT_URL: "https://test.sourcingjournal.pmcqa.com"
+          # HASH_URL: "https://test.sourcingjournal.pmcqa.com/wp-content-vipgo-sites/test/themes/vip/pmc-sourcingjournal-2021/version-hash"
+          #
+          # main branch example:
+          # ENVIRONMENT_URL: "https://sourcingjournal.pmcqa.com"
+          # HASH_URL: "https://sourcingjournal.pmcqa.com/wp-content-vipgo/themes/vip/pmc-sourcingjournal-2021/version-hash"
+          #
```

Comment:
As this is a public repo, these references should be removed.

Reply:
Done in [fec56b0](https://github.com/penske-media-corp/github-workflows-wordpress/pull/53/commits/fec56b082785dc57de430439ad320367bfd120c1)